sample.bluetooth.peripheral_hr build fails on rv32m1_vega_ri5cy

Recording this as an official issue since it's starting to show up in PR builds:

samples/bluetooth/peripheral_hr/sample.bluetooth.peripheral_hr_rv32m1_vega_ri5cy in  rv32m1_vega_ri5cy:samples/bluetooth/peripheral_hr/sample.bluetooth fails CI as demonstrated at the first failure of https://buildkite.com/zephyr/zephyr-daily/builds/190.

```
/workdir/zephyr/subsys/bluetooth/controller/hci/hci.c:32:10: fatal error: lll_sync_iso.h: No such file or directory
   32 | #include "lll_sync_iso.h"
      |          ^~~~~~~~~~~~~~~~
```

This will be fixed by #31005 when that gets merged.

--- a/subsys/bluetooth/controller/ll_sw/lll_sync_iso.h
+++ b/subsys/bluetooth/controller/ll_sw/lll_sync_iso.h
@@ -6,5 +6,4 @@
 
 struct lll_sync_iso {
 	struct lll_hdr hdr;
-	uint8_t is_enabled:1;
 };

--- a/subsys/bluetooth/controller/ll_sw/lll_sync_iso.h
+++ b/subsys/bluetooth/controller/ll_sw/lll_sync_iso.h
@@ -4,7 +4,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 struct lll_sync_iso {
 	struct lll_hdr hdr;
 	uint8_t is_enabled:1;

--- a/subsys/bluetooth/controller/ll_sw/ull_sync_iso.c
+++ b/subsys/bluetooth/controller/ll_sw/ull_sync_iso.c
@@ -92,7 +92,6 @@ uint8_t ll_big_sync_create(uint8_t big_handle, uint16_t sync_handle,
 	/* Initialise ULL and LLL headers */
 	ull_hdr_init(&sync_iso->ull);
 	lll_hdr_init(lll_sync, sync);
-	lll_sync->is_enabled = true;
 
 	return BT_HCI_ERR_SUCCESS;
 }
@@ -109,11 +108,6 @@ uint8_t ll_big_sync_terminate(uint8_t big_handle)
 		return BT_HCI_ERR_UNKNOWN_ADV_IDENTIFIER;
 	}
 
-	if (!sync_iso->lll.is_enabled) {
-		return BT_HCI_ERR_CMD_DISALLOWED;
-	}
-
-	/* TODO: Stop ticker */
 	err = ull_ticker_stop_with_mark(
 		TICKER_ID_SCAN_SYNC_ISO_BASE + big_handle,
 		sync_iso, &sync_iso->lll);
@@ -128,7 +122,6 @@ uint8_t ll_big_sync_terminate(uint8_t big_handle)
 	big_sync_estab = sync_iso->node_rx_estab.link;
 	ll_rx_link_release(big_sync_estab);
 
-	sync_iso->lll.is_enabled = false;
 	ull_sync_iso_release(sync_iso);
 
 	return BT_HCI_ERR_SUCCESS;

